suitcase/hdf5.py

```diff
@@ -94,8 +95,27 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 compression='gzip',
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
-                    dataset = data_group.create_dataset(
-                        key, data=data, compression='gzip', fletcher32=True)
+                    data = np.array(data)
+                    try:
+                        # save numerical data
+                        dataset = data_group.create_dataset(
+                            key, data=data, compression='gzip', fletcher32=True)
+                    except TypeError:
+                        try:
+                            # save data with str type, or list of str
+                            if len(data.shape) == 1:
+                                data_len = len(data[0])
+                            else:
```

Comment:
better to say `elif data.ndim == 2` and then add an `else` that raises an error saying "we can't go past 2"